ledger/src/snapshot_utils.rs

```diff
@@ -101,6 +106,35 @@ pub fn package_snapshot<P: AsRef<Path>, Q: AsRef<Path>>(
     Ok(package)
 }
 
+pub fn serialize_status_cache(
+    slot_deltas: &[SlotDelta<TransactionResult<()>>],
+    snapshot_links: &TempDir,
+) -> Result<()> {
+    // the status cache is stored as snapshot_path/status_cache
+    let snapshot_status_cache_file_path =
+        snapshot_links.path().join(SNAPSHOT_STATUS_CACHE_FILE_NAME);
+
+    let mut status_cache_serialize = Measure::start("status_cache_serialize-ms");
+    let consumed_size = serialize_snapshot_data_file(
+        &snapshot_status_cache_file_path,
+        MAX_SNAPSHOT_DATA_FILE_SIZE,
+        |stream| {
+            serialize_into(stream, slot_deltas)?;
+            Ok(())
+        },
+    )?;
+    status_cache_serialize.stop();
+
+    datapoint_info!("snapshot-status-cache-file", ("size", consumed_size, i64));
```

Comment:
Tidying up these `data_point_info!`s by making serialize_snapshot_data_file handle them might be a good idea?

Reply:
Sure, why not

Reply:
done! 51d786a44e587a9a8a690d032341dfbff836f96d